components/Header

```diff
@@ -0,0 +1,15 @@
+import { Component } from 'react';
+import { Header } from 'react-native-elements';
+import { Icon } from 'react-native-vector-icons/FontAwesome';
+
+export default class AppHeader extends Component {
+    render() {
+        return (
+            <Header
+                leftComponent={{ icon: 'chevron-left', onPress: () => console.log('pressed'), color: '#000', size: 40 }}
+                centerComponent={{ text: 'header', style: { color: '#000', fontSize: 30, fontWeight: '500' } }}
```

Comment:
Sorry I should've mentioned this earlier. Can you convert this class into a stateless functional component (SFC)? It also needs to take in props which for now can just be the header text. Make sure you import the propTypes library so that you can specify the props that should be passed in. A good example of a SFC and propTypes can be found here https://github.com/CodetheChangeFoundation/dda/blob/master/components/auth/SignInForm.js.